arbnode/seq_coordinator.go

```diff
@@ -865,7 +865,33 @@ func (c *SeqCoordinator) Start(ctxIn context.Context) {
 				err, "newRedisUrl", c.config.NewRedisUrl)
 		}
 	}
-	c.CallIteratively(func(ctx context.Context) time.Duration { return c.chooseRedisAndUpdate(ctx, newRedisCoordinator) })
```

Comment:
coordinator should, at most, stop trying if it gets a specific error from transaction_streamer
like delayed-sequencer, it's not expected to ever really be enabled on a node with syncTillBlocl

Reply:
Done.